backend/app/notification/email/providers/__init__.py

```
@@ -0,0 +1 @@
+from .brevo import BrevoEmailProvider
```

Ruff (0.8.0)

`.brevo.BrevoEmailProvider` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)